src/commands.ts

```diff
@@ -289,14 +291,37 @@ export function findText(args: FindTextArgs) {
 		return;
 	}
 
+	// Find the next index given a regex
+	const regexIndexOf = (str: string, regex: string, pos?: number | undefined) => {
+		pos = pos || 0;
+
+		// Search forward from position in string
+		let indexOf = str.substring(pos).search(RegExp(regex));
+
+		// Add position to found index to account for beginning of string
+		return (indexOf >= 0) ? (indexOf + pos) : indexOf;
+	}
+
+	// Find the previous index given a regex
+	const regexLastIndexOf = (str: string, regex: string, pos?: number | undefined) => {
+		pos = pos || str.length;
+
+		// Reverse the string and search for first occurrence
+		let strReversed = str.substring(0, pos).split("").reverse().join("");
```

Comment:
That's a good point. I was trying to be too clever and scan the entire string, but it is probably cheaper to do so in this case anyway since the strings are short.

I'll make the change and then test with some multi-character patterns. All of my initial tests were a single character, which is why I didn't hit the problem.